lib/vmdb/util.rb

```diff
@@ -1,7 +1,10 @@
 module VMDB
   module Util
-    def self.http_proxy_uri
-      proxy = VMDB::Config.new("vmdb").config[:http_proxy] || {}
+    def self.http_proxy_uri(proxy_config = :default)
+      # TODO: (julian) This looks really messy. Need to make it look nicer.
+      vmdb_proxy = VMDB::Config.new("vmdb").config[:http_proxy]
```

Comment:
The problem with `vmdb_proxy = Settings.http_proxy` is the tests no longer pass. https://github.com/ManageIQ/manageiq/pull/10368/commits/73126d9c379b5c82f38977a80b336f08a59baf00

As in the tests we pass in a new options hash, which when the code is `VMDB::Config.new("vmdv").config[:http_proxy]` works great, but with ` Settings.http_proxy` we can't stub the `settings.yml` that way.